webpack.config.js

```diff
@@ -1,25 +1,49 @@
-var path = require('path');
-var SRC_DIR = path.join(__dirname, '/client/src');
-var DIST_DIR = path.join(__dirname, '/client/dist');
-// const HtmlWebpackPlugin = require('html-webpack-plugin');
-// const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
```

Comment:
Does this work for you?  I had it in mine, but every time webpack ran it added a <script> tag to the html even when there was already one there.  I probably had it set up wrong.

Reply:
I think it does now but I have done so many iterations of webpack , I really am not sure. I will confirm this to you today/tomorrow by removing my html from dist